devices/Tm1637/app.config

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
please remove all app.config, they are not necessary and create more mess than they help